app/views/mailer/owners_update.html.erb

```diff
@@ -0,0 +1,44 @@
+<% @title = t(".title") %>
+<% @sub_title = t(".subtitle", user_handle: @user.handle) %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <br>
+        <p>
+          <%= @owner.id == @user.id ? I18n.t("mailer.owners_update.body_self", status: @status, gem: @rubygem.name) : I18n.t("mailer.owners_update.body_others", status: @status, gem: @rubygem.name, owner_handle: @owner.handle) %>
+        </p>
+        <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+        <p>
+          Update your notification settings if you want to stop receiving owner change emails
+        </p>
+        <p>If this change is expected, you do not need to take further action.</p>
+        <p>
+          <strong>Only if this change is unexpected</strong>
+          please take immediate steps to secure your account and gems:
```

Comment:
Can you please create a separate PR to change this to partial? We seem to be using it at quite a few places.

Reply:
https://github.com/rubygems/rubygems.org/pull/2421